### PR TITLE
Upgraded sigh to the new fastlane UI

### PR DESCRIPTION
**Pull request by KrauseFx**

### CODE REVIEW

File: lib/sigh/runner.rb [hunk 1]
```diff
@@ -11,35 +11,35 @@ def run
...
       Spaceship.select_team
-      Helper.log.info "Successfully logged in"
+      UI.message "Successfully logged in"
 
       profiles = [] if Sigh.config[:skip_fetch_profiles]
       profiles ||= fetch_profiles # download the profile if it's there
 
       if profiles.count > 0
-        Helper.log.info "Found #{profiles.count} matching profile(s)".yellow
+        UI.success "Found #{profiles.count} matching profile(s)"
```
Comment (asfalcone) [inline lib/sigh/runner.rb]: is the change from `.yellow` to `success` here intentional?

File: lib/sigh/runner.rb [hunk 2]
```diff
@@ -137,11 +137,11 @@ def certificate_to_use
...
 
       if certificates.count > 1 and !Sigh.config[:development]
-        Helper.log.info "Found more than one code signing identity. Choosing the first one. Check out `sigh --help` to see all available options.".yellow
-        Helper.log.info "Available Code Signing Identities for current filters:".green
+        UI.important "Found more than one code signing identity. Choosing the first one. Check out `sigh --help` to see all available options."
+        UI.important "Available Code Signing Identities for current filters:"
         certificates.each do |c|
           str = ["\t- Name:", c.owner_name, "- ID:", c.id + "- Expires", c.expires.strftime("%d/%m/%Y")].join(" ")
-          Helper.log.info str.green
+          UI.message str.green
```
Comment (asfalcone) [inline lib/sigh/runner.rb]: Should this be `UI.success` if it needs to be green?

**Comment by asfalcone** [comment]
Left 2 comments. Looks good overall!

File: lib/sigh/runner.rb (hunk 2 above)
Comment (KrauseFx) [inline lib/sigh/runner.rb]: It's not really success, it's just a green status, so I thought it's cleaner to use a message as a level, but still have a green text

File: lib/sigh/runner.rb (hunk 1 above)
Comment (KrauseFx) [inline lib/sigh/runner.rb]: Yeah :+1:

**Comment by KrauseFx** [comment]
Awesome, thanks for reviewing @asfalcone :+1:

**Pull request merged**